mypy/checkstrformat.py

```diff
@@ -55,7 +55,7 @@ def __init__(self,
         self.exprchk = exprchk
         self.msg = msg
 
-    def check_str_interpolation(self, str: StrExpr, replacements: Node) -> Type:
+    def check_str_interpolation(self, str: Union[StrExpr, BytesExpr, UnicodeExpr], replacements: Node) -> Type:
         """Check the types of the 'replacements' in a string interpolation
         expression: str % replacements
         """
```

Comment:
Is the bytes formatting in 3.5 a TODO? Or is it something that I'll need to check for in code?